src/garage/tf/algos/rl2.py

```diff
@@ -310,11 +307,11 @@ class RL2(MetaRLAlgorithm, abc.ABC):
 
     """
 
-    def __init__(self, rl2_max_episode_length, meta_batch_size, task_sampler,
-                 meta_evaluator, n_epochs_per_eval, **inner_algo_args):
+    def __init__(self, meta_batch_size, task_sampler, meta_evaluator,
+                 n_epochs_per_eval, **inner_algo_args):
         self._inner_algo = RL2NPO(**inner_algo_args)
-        self._rl2_max_episode_length = rl2_max_episode_length
         self.env_spec = self._inner_algo._env_spec
+        self._rl2_max_episode_length = self.env_spec.max_episode_length
```

Comment:
```suggestion
        self._rl2_max_episode_length = self._env_spec.max_episode_length * episodes_per_trial
```

Reply:
I think it should be `self.env_spec.max_episode_length`, while the inner algorithm should take `self._env_spec.max_episode_length * episodes_per_trial`.

Reply:
@ahtsan right, that's how it was initially.